fli-iam/shanoir-ng
# shanoir-issue#2539: use getType instead of instanceof to check modality

How to test:
Create studyA
Create studyB
Import dicom in studyA
Open solr > select the imported datasets > copy to studyB
Open studyB > subject tab > edit > remove subjectA from studyB > save
Open studyA > subject tab > edit > remove subjectA from studyA > save

Expected result: datasets are all removed from solr

Tested on neurinfo-qualif and working OK
